cmd/wallet-web/src/App.vue

```diff
@@ -54,7 +54,7 @@ onUnmounted(() => {
 <template>
   <div class="w-screen h-screen font-sans bg-neutrals-mischka">
     <div v-if="!loaded" class="flex relative top-1/3 flex-col justify-start items-center">
-      <IconSpinner />
+      <spinner-icon />
```

Comment:
Can we use camel case for custom-defined components?
```suggestion
      <SpinnerIcon />
```

Reply:
Sure, I can change that, and that would go for all the components? 
Is there a reason why CamelCase is preferred here? I think before my changes, most were in kebab-case..

Reply:
From discussing with Anton, using PascalCase will make it more clear that it is a custom defined component, and also using kebab-case might make the template tags longer (ex `credential-details-banner-component`), which I agree with. @rolsonquadras @talwinder50 Any thoughts or can I go ahead with the PascalCase?

Reply:
we can go head with PascalCase @HeidiHan0000

Reply:
Changing to PascalCase...